src/Tools/BuildUtil/BuildUtil/WriteCodeFragmentEx.cs

```diff
@@ -21,7 +21,7 @@ public class WriteCodeFragmentEx : Task
 
         public override bool Execute()
```

Comment:
@sharwell filed the following to track the two big cleanup items. Will likely do before I leave today. 

https://github.com/dotnet/roslyn/issues/20526